### DISCUSSION TITLE
Images do not show

### DISCUSSION THREAD

**Discussion by ghost** [nunocoracao/blowfish#145] [Q&A] [answered]
Images do not show in both home page and articles.

For example, in `/config/_default/params.toml`, I set `homepage.homepageImage = "/img/takodachi_wallpaper.png"` where `config` and `img` are subdirectories under the root directory.  However, the images just never show:

<img width="1111" alt="Screenshot 2022-10-19 at 10 50 36 PM" src="https://user-images.githubusercontent.com/88360485/196725621-fbf4b01a-20fc-4907-b422-50551d2f6f42.png">

What might the problem possibly be?

**Comment by nunocoracao** [comment]
Hello @luma0724 do you have a repo I can take a look at in order to help. The images should be places in the assets folder (or static) not sure if you are already doing that.

**Reply by ghost** [reply]
@nunocoracao 
This is now the deployed website: https://spcccomputerclub.github.io/
and the repo: https://github.com/spcccomputerclub/spcccomputerclub.github.io

**Reply by ghost** [reply]
I have tried placing the images in assets, but it does not work...

**Comment by nunocoracao** [comment, marked as answer]
There was a bug in the code which was fixed in 
https://github.com/nunocoracao/blowfish/issues/148

I've also opened a PR to your code to upgrade the theme and put the image in the right folder to confirm that it works
https://github.com/spcccomputerclub/spcccomputerclub.github.io/pull/1

**Comment by spcccomputerclub** [comment]
Hi @nunocoracao Sorry for bothering you again, but my issue is still unfixed. You might want to refer back to my repo: https://github.com/spcccomputerclub/spcccomputerclub.github.io

Despite all those messy commits, the `card.html`, `hero.html` and `background.html` are already updated the lines of code added in #148:

```
{{ with .Site.Params.homepage.homepageImage }}
{{ $homepageImage := resources.Get . }}
{{ if $homepageImage }}
...
{{ end }}
{{ end }}
```
and my homepage image is correctly placed under `/assets/img/`, but it still doesn't work:

<img width="1020" alt="Screenshot 2022-10-20 at 10 50 04 PM" src="https://user-images.githubusercontent.com/113744670/196982553-274d998c-1299-4321-b2a7-909d7da7d423.png">

So the steps to reproduce it:
1. Upload the image in `/assets/img/`
2. Set `homepage.homepageImage = "/assets/img/<image_name>"` and `homepage.layout = "hero"`

Expected: well the image should be loaded after the fix

**Reply by nunocoracao** [reply]
homepage.homepageImage = "/img/<image_name>" try removing the /assets/ part, otherwise I can have a looker later today

**Reply by nunocoracao** [reply]
Hey @spcccomputerclub the theme installation was all messed up. I've removed it as reinstalled it as a git submodule which will make it easier to manage. My initial pull request had the new version of blowfish already so not sure what happened. Also fixed the path to the image. it's working on this PR
https://github.com/spcccomputerclub/spcccomputerclub.github.io/pull/2

**Reply by spcccomputerclub** [reply]
Thx, everything works now:D
